pkgs/applications/radio/abracadabra/default.nix

```diff
@@ -6,13 +6,13 @@
 
 stdenv.mkDerivation rec {
   pname = "abracadabra";
-  version = "2.2.4";
+  version = "2.3.1";
 
   src = fetchFromGitHub {
     owner = "KejPi";
     repo = "AbracaDABra";
     rev = "v${version}";
-    sha256 = "sha256-gpZ6ckV//fhDlpAqmMwL4XNXX7xFmGi58fkOC4oRcDM=";
+    sha256 = "sha256-HRwgtNCrEzo6uXNJHG3PwpBAXVTppYUTmiwKWi3reho=";
```

Comment:
```suggestion
    hash = "sha256-HRwgtNCrEzo6uXNJHG3PwpBAXVTppYUTmiwKWi3reho=";
```

Reply:
This is a change that should be done once a tree-wide basis via a script...

Reply:
I am not aware of tree-wide initiative for this, right now, it seems to be incremental each time a line that contains sha256 is updated.

Reply:
`grep -R 'sha256 = "sha256-' pkgs  | wc -l` yields 7356 results. My point is that it simply should be done with one big search-and-replace at some point.

Reply:
I think the reason it hasn't been done because not all instances can be replaced ( eg when its not passed directly into fetcher like fetchulr or fetchpatch )

As for gradual replacement,  much like by-name migration is done to when there is another reason to make change to derivation, same for sha256, if that line is changed, might as well migrate it to `hash`